influxdb/src/main/scala/akka/stream/alpakka/influxdb/impl/AlpakkaResultMapperHelper.scala

```diff
@@ -63,7 +63,7 @@ private[impl] class AlpakkaResultMapperHelper {
         val columnName: String = column.name()
         val fieldType: Class[_] = field.getType()
 
-        if (!field.isAccessible()) {
+        if (!field.canAccess(model)) {
```

Comment:
Not sure if this is relevant here, but I don't think `canAccess` exists in Java 8.

Reply:
Ah, you're right. I'll silence this warning.